src/components/Header.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { NavLink } from "react-router-dom"
```

Comment:
If not using data remove it Navlink from react-router-dom. Not sure what your syntax was on NavLink but check out this site https://v5.reactrouter.com/web/api/NavLink/isactive-func

Reply:
The assets folder is under the src directory from the original push. And when importing the mockCat.js file with the variable mockCat, I believe you wish for us to display the entire array in the display portion of App.js , is that correct? Thank you in advance!

Reply:
Oh wait! The assets folder is not in the original push, sorry for that false statement.